saleor/graphql/core/enums.py

```diff
@@ -78,6 +79,11 @@ def to_enum(enum_cls, *, type_name=None, **options) -> graphene.Enum:
     return graphene.Enum(type_name, enum_data, **options)
 
 
+LanguageCodeEnum = graphene.Enum(
+    "LanguageCodeEnum",
```

Comment:
https://github.com/mirumee/saleor/pull/6822 in this PR we want to remove suffix `Enum`, maybe you could change it to keep consistency in enum names?

Reply:
@d-wysocki  I just tried to do that, then I realized that the enum `LanguageCodeEnum` is used also in the translation module. Renaming it will force me to rename all naming in translations which cause bigger mess in the PR than it is now :D 
Also it will cause a lot of conflicts in 6822, so I would prefer to leave it as it is now, and rename them in PR dedicated to the enums.